spec/requests/apis/v1/job_categories_api_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'API categorias de trabalho' do
+  context 'GET /api/v1/job_categories' do
+    it 'com sucesso' do
+      create(:job_category, name: 'Full Stack')
+      create(:job_category, name: 'Web Design')
+      create(:job_category, name: 'Ruby on Rails')
+
+      get '/api/v1/job_categories'
+
+      expect(response.status).to eq 200
+      expect(response.content_type).to include 'application/json'
+      json_response = JSON.parse(response.body)
+      expect(json_response.class).to eq Array
+      expect(json_response.count).to eq 3
+      expect(json_response.first.keys).not_to include 'created_at'
+      expect(json_response.first.keys).not_to include 'updated_at'
+      expect(json_response.first['id']).to eq 1
+      expect(json_response.first['name']).to eq 'Full Stack'
+      expect(json_response.second['id']).to eq 2
+      expect(json_response.second['name']).to eq 'Web Design'
+      expect(json_response.third['id']).to eq 3
+      expect(json_response.third['name']).to eq 'Ruby on Rails'
+    end
+
+    it 'retorna um aviso de que não existem categorias cadastradas' do
```

Comment:
avaliem se um retorno de um array vazio não é mais interessante nesse caso

Reply:
Boa @hreis1. Vou mudar o status para um `204 No Content` e retornar uma array vazia.

Reply:
O status `204 No Content` força um retorno com zero bits (ou seja, `nil`). Então vamos seguir com uma resposta `200` e a Array vazia :)